exercises/string-manipulation-2/src/last-chars.js

```diff
@@ -0,0 +1,10 @@
+/* exported lastChars */
+function lastChars(length, string) {
+  var newString = '';
+  for (var i = string.length - length; i < string.length; i++) {
+    newString += string[i];
+  } if (length > string.length) {
+    return string;
+  }
```

Comment:
I would do this conditional as the first part of the function, so that if it is this special case, you don't do all the normal work in addition to this special piece of work.